intro-to-git/collaborating.md

```diff
@@ -0,0 +1,171 @@
+# Collaborating
+
+## Learning Goals
+
+- Explain how git is version control system used to collaborate on code
+- Compare the difference between code on a local machine versus code on a remote repository
+- Explain how to get changes from a remote repository using git pull (or git fetch and git merge)
+- Explain how to push changes to a remote repository using git push
+- Recall that a local machine must merge in changes from the remote before pushing
+
+## Introduction
+
+Developers working on projects tracked by Git build their projects with commits. We can imagine a single developer, Nakita, working on her project named Hike Planner. By the end of the week, she's created twenty commits, and her Git history tells her the story of the code.
+
+How would Nakita's workflow change if the Hike Planner team grows? Multiple developers can work on a project, and Git has even more tools to facilitate collaborating. The key to collaborating with Git is working to converge on a single, unified Git history together.
+
+
+## Vocabulary and Synonyms
+
+| Vocab | Definition | Synonyms | How to Use in a Sentence
+| --- | --- | --- | ---
+Remote repository | asdf | Remote, remote repo, `origin` | asdf
+
+## Collaboration Means Looking at the Same Source of Code
+
+Nakita's friends, Frida and Raul, want to join Nakita and help her build Hike Planner. How does Nakita share the current Git history that's already built? How do Frida and Raul get access to this Git history? Can Frida and Raul work at the same time as Nakita? Do Frida and Raul make commits too? How do all of these commits get received? How do they get shared?
+
+Git is highly configurable, so most of these settings can change. However, in the vast majority of cases, Git enables collaboration with this setup:
+
+- The project should be hosted somewhere, where team members have access. The project will can be called the project "repo."
+- Each developer will download the repo onto their machine using the Git command `clone`. This "download" will carry the Git history
+- All team members should contribute code changes by making commits on their own machines
+- All team members will receive commits and updates from the main hosted repo
+- All team members will send commits and updates to the main hosted repo
```

Comment:
Change `hosted repo` to `remote repo` for consistent vocabulary.

Reply:
```suggestion
- All team members will receive commits and updates from the main remote repo
- All team members will send commits and updates to the main remote repo
```

Tagging @cwboden for visibility!

Carson, if you're interested in some changes including you as a co-author to the commit, feel free to use the suggestion feature!

Reply:
Also hmmmm. "Remote" doesn't get defined until the next section, so that's a thought I have. HMmm.